rules/ifupdown2.mk_

```diff
@@ -0,0 +1,11 @@
+# ifupdown2 package
```

Comment:
Added by accident?

Reply:
Ah yes. It comes from another branch work. Thanks for the feedback. I will rebase and do a git clean before.